first_ruby/app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_tree ../images
```

Comment:
Whats this file for?

Reply:
This file is to locate and load tasks which can be run from the command line. Which also known to help building, packaging and testing of Rails code.

Reply:
Set what file is going to get pre-compiled by sprockets rails which is a Asset pipeline to convert Sass file to css.

Reply:
oh? This is to convert SASS file to css? Then why the first line is ../images (which holds image files?

Reply:
This file is to specify the directory that is managed by the *Asset Pipeline*,  which is inside the `asset/` directory, name the folder accordingly as `images` which contains image files , `javascripts` which contain javascript files and `stylesheets` which contain css or scss files.